src/main/java/ua/ostapenko/hw10/Truck.java

```diff
@@ -0,0 +1,22 @@
+package ua.ostapenko.hw10;
+
+public class Truck extends Car {
```

Comment:
I can't find methods to cargo loading and unloading

Reply:
unloadCargo and loadCargo methods are added